src/components/PlanRoom/RoomTables/Item.tsx

```diff
@@ -0,0 +1,184 @@
+import { useCallback, useState, SetStateAction } from 'react';
+import { Box, Textarea } from '@chakra-ui/react';
+import update from 'immutability-helper';
+import { useDrop, XYCoord, useDrag } from 'react-dnd';
+import ItemTypes from './ItemTypes';
+import { TypeItem, TableType } from './Types';
+import SaveBoard from '../SaveRoomTables/SaveBoard';
+
+let changeValueTable: Function;
+let changeValueTableBorderRadius: Function;
+let newTable: Function;
+let save: Function;
+const backgroundColor: string = '#211F1F';
+
+const Items = () => {
+  const [tables, setTables] = useState<{
+    [key: string]: { top: number; left: number; text: string; width: number; height: number; borderRadius: number };
```

Comment:
You should create separate `type` definition and store it in proper file outside of this component.